src/main/java/Lovie.java

```diff
@@ -0,0 +1,214 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Lovie {
+    public static void main(String[] args) {
+        Scanner inputGetter = new Scanner(System.in);
+        String input;
+        ArrayList<Task> tasksList = new ArrayList<Task>();
+
+        String LOGO = "██╗░░░░░░█████╗░██╗░░░██╗██╗███████╗ \n" +
+                "██║░░░░░██╔══██╗██║░░░██║██║██╔════╝ \n" +
+                "██║░░░░░██║░░██║╚██╗░██╔╝██║█████╗░░ \n" +
+                "██║░░░░░██║░░██║░╚████╔╝░██║██╔══╝░░ \n" +
+                "███████╗╚█████╔╝░░╚██╔╝░░██║███████╗ \n" +
+                "╚══════╝░╚════╝░░░░╚═╝░░░╚═╝╚══════╝ \n";
+        print(LOGO + "Hey hey! My name is Lovie! How can I help you today?");
+
+        while (true) {
+            System.out.print("\t");
+            input = inputGetter.nextLine();
+            String processedInput = input.toLowerCase().trim();
+
+            // Switch statement to keep track of user input commands
+            switch (processedInput) {
```

Comment:
This switch block is very long. Consider abstracting further to break this down.